### ISSUE TITLE
Dynamic-DeepHit: A Deep Learning Approach for Dynamic Survival Analysis With Competing Risks Based on Longitudinal Data

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#3]
## TL;DR
This paper proposes a deep neural network model of competing risk events with longitudinal data. 

### Paper Link

https://ieeexplore.ieee.org/document/8681104

### Author/Institution

Chang hee Lee (Chung-Ang University), Jinsung Yoon (University of California), Mihaela van der Schaar (University of Cambridge, University of California)

## Overview

This paper develops a novel deep learning approach that is able to successfully address current limitations of standard statistical approaches such as land marking and joint modeling. Dynamic-DeepHit flexibly incorporates the available longitudinal data comprising various repeated measurements (rather than only the last available measurements) in order to issue dynamically updated survival predictions for one or multiple competing risk(s).

## Methods

Time measurement, Diseases, Biomarkers, Biological system modeling, Deep learning, Computational modeling

## Cite

Lee C, Yoon J, Schaar MV. Dynamic-DeepHit: A Deep Learning Approach for Dynamic Survival Analysis With Competing Risks Based on Longitudinal Data. IEEE Trans Biomed Eng. 2020 Jan;67(1):122-133. doi: 10.1109/TBME.2019.2909027. Epub 2019 Apr 3. PMID: 30951460.

## Comments

Python code available:
